dev/io.openliberty.microprofile.telemetry.1.0.internal/src/io/openliberty/microprofile/telemetry/internal/cdi/OpenTelemetryProducer.java

```diff
@@ -95,6 +100,11 @@ public OpenTelemetry getOpenTelemetry() {
         //By default, MicroProfile Telemetry tracing is off.
         //The absence of an installed SDK is a “no-op” API
         //Operations on a Tracer, or on Spans have no side effects and do nothing
+        if (!warned) {
+            String msg = Tr.formatMessage(tc, "CWMOT6000.tracing.is.disabled");
```

Comment:
We've added the application name to the message, try using this to get the application name:
```java
ComponentMetaDataAccessorImpl.getComponentMetadata().getJ2EEName().getApplication();
```